### DISCUSSION TITLE
CSS insertion not working

### DISCUSSION THREAD

**Discussion by sub0ne** [requarks/wiki#3638] [Error / Bug Report]
**Describe the bug**
I added some CSS in the insertion block, but this  is not working.

**To Reproduce**
1. Go to Administration, Theme
2. Add something like:
       .v-list-item {min-height: 0px;}

**Expected behavior**
Toc should be more dense now

**Screenshots**

**Host Info (please complete the following information):**
 - OS: [Ubuntu 18.04]
 - Wiki.js version: [2.1.113]
 - Database engine: [mysql 5.7]

Even it would be nice to have a tutorial of how to customize the UI, as there is no documentation yet.

**Comment by Smankusors** [comment]
try to add `!important` to the css value?
